samples/cooperative-cancellation/sample.md

```diff
@@ -1,23 +1,24 @@
 ---
 title: Cooperative cancellation
 summary: Stopping both the endpoint and a long running handler with a cancellation token
-reviewed: 2021-08-31
+reviewed: 2024-01-20
 component: Core
 ---
 
-
 Run the solution. A single console application starts up: `Server`.
 
 ## Code walk-through
 
-When the endpoint is started a `CancellationToken` is passed in from a `CancellationTokenSource`
+When the endpoint is started, a message is sent to the endpoint, which triggers a long-running message handler that enters a loop simulating a task that takes 3 seconds to execute, like writing a bunch of records to a database, by calling the async `Task.Delay()` operation.
```

Comment:
Not quite accurate, right? You've made it take infinite time to execute, but just log something every 3 seconds.